SUPPORTED-FORMATS.md

```diff
@@ -120,3 +120,4 @@ the [Analysis Parsers Library](https://github.com/jenkinsci/analysis-model/).
 | 100 | yamllint | yamlLint() | - - | [YamlLint](https://yamllint.readthedocs.io/) |  |
 | 101 | yui | yuiCompressor() | - - | YUI Compressor |  |
 | 102 | zptlint | zptLint() | - - | ZPT Lint |  |
+| 103 | dscanner | dscanner() | - - | DScanner | **/dscanner-report.json |
```

Comment:
Do not edit this file, run the tests in `ToolsLister` to auto-genenerate.

Reply:
Done